rust/Cargo.toml

```diff
@@ -9,7 +9,7 @@ keywords = ["geography", "geospatial", "gis", "gps", "olc"]
 exclude = ["rust.iml"]
 
 [dependencies]
-geo = "^0.4.3"
+geo = "0.16.0"
```

Comment:
This is effectively dropping support for versions < 0.16.0, right? Can you add some description of why that's desirable?

Reply:
Rust uses a slightly weird variant of semver so versions `0.4.x`, `0.5.x`, .... `0.16.x`, `1.x.y`, `2.x.y` are all mutually incompatible (you can see more details [here](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html)). The net result is that previously the library only supported versions 0.4.x and after this change will support only 0.16.x.